crates/cli/src/report.jinja

```diff
@@ -0,0 +1,298 @@
+                 }
+
+                 if (shouldShow) visibleCount++;
+             });
+
+             // Update counter
+             document.getElementById('visible-benchmarks').textContent =
+                 `Showing ${visibleCount} of ${rows.length} benchmarks`;
+         }
+
+         // Format numbers with commas using toLocaleString()
+         function formatCycleCounts() {
+             document.querySelectorAll('.cycle-count').forEach(element => {
+                 const num = parseInt(element.textContent.trim());
+                 if (!isNaN(num)) {
+                     element.textContent = num.toLocaleString();
+                 }
+             });
+         }
+
+         // Initialize filters and formatting on page load
+         document.addEventListener('DOMContentLoaded', function() {
+             updateFilters();
+             formatCycleCounts();
+         });
+        </script>
+
+        <div id="effect-size-methodology" style="margin-top: 40px; padding: 20px; background: #f9f9f9; border-radius: 5px;">
+            <h2>Statistical Methodology</h2>
+            <p>
+                This report follows the <a href="https://github.com/bytecodealliance/rfcs/blob/main/accepted/benchmark-suite.md" target="_blank">Bytecode Alliance RFC standards</a>
+                for statistically rigorous performance analysis, using <strong>effect size analysis</strong> with confidence intervals
+                rather than simple averages or visual comparisons.
+            </p>
+            <p>
+                <strong>Statistical Foundation:</strong> Built on Student's t-test principles for comparing two means, with Behrens-Fisher analysis
+                for unequal variances. This provides confidence intervals like "95% confident that engine A is 1.05x to 1.15x faster than engine B"
+                or "there is no statistically significant difference in performance between the engines".
+            </p>
+            <p><strong>This approach quantifies both:</strong></p>
+            <ul>
+                <li><strong>Statistical significance</strong>: Whether the performance difference is likely real (not due to measurement noise)</li>
+                <li><strong>Practical significance</strong>: The magnitude of the performance difference with confidence bounds</li>
+            </ul>
+            <p>
+                <strong>Key Metrics:</strong> Analysis focuses on execution cycles as the primary metric, with coefficient of variation (CV)
+                used to assess measurement reliability. Tests with CV > 5% are flagged as potentially noisy.
+            </p>
+            <p>
+                <strong>References:</strong>
+                <a href="https://github.com/bytecodealliance/rfcs/blob/main/accepted/benchmark-suite.md" target="_blank">Benchmark Suite RFC</a> |
+                <a href="https://github.com/bytecodealliance/sightglass#benchmarking-with-sightglass" target="_blank">Sightglass Documentation</a>
```

Comment:
Throughout the document, don't do the `target="_blank"` stuff; just make them be normal links.